### ISSUE TITLE
A typo in .busted makes the file silently ignored

### ISSUE THREAD

**Issue by mkpankov** [Olivine-Labs/busted#597]
It makes it hard to understand why options from `.busted` are ignored.

**Comment by Tieske** [comment]
what typos? can you provide a minimal example?

**Comment by mkpankov** [comment]
Basically any typo:
```
➜  test_suites git:(lua-tests) ✗ cat .busted
return {
  _all = {
    coverage = tru
  }
}
➜  test_suites git:(lua-tests) ✗ busted .
●
1 success / 0 failures / 0 errors / 0 pending : 0.003184 seconds
➜  test_suites git:(lua-tests) ✗ ls luacov.stats.out
ls: cannot access 'luacov.stats.out': No such file or directory
```

My main concern is absence of error/warning from busted when `.busted` is invalid syntactically.

**Comment by Tieske** [comment]
thing is in this case; it is syntactically correct. In this case you're accessing a global named `tru`. Since it's undefined, the result is `nil`, and hence it is falsy, and no coverage report is ran.

**Comment by mkpankov** [comment]
Well, real syntax error isn't reported too:
```
return {
  _all = {
    coverage = false
  }

```

**Comment by Tieske** [comment]
fix in #600
